tests/assets/move-projects/move-basics/scripts/Loops.move

```diff
@@ -15,3 +15,15 @@ script {
         }
     }
 }
+
+script {
+    use std::vector;
+    fun empty_loop_param_with_a_vector(a: vector<u64>) {
+        vector::pop_back(&mut a);
+        let s = vector::pop_back(&mut a);
```

Comment:
What does this second line with `vector::pop_back`? :upside_down_face: is it possible to combine it with the first one?

Reply:
Well, this is not a meaningful script in any way 😄 - the purpose of the script is:
 - have a script that has a vector for the input parameter
 - make sure the script will fail in the case `vector.len() < 2`
 
I can append a comment like this in the script in the next PR, regarding:

> What does this second line with vector::pop_back? 🙃 is it possible to combine it with the first one?

Feel free to suggest anything here but consider the two points above :D